RollbarNotifier/Sources/RollbarNotifier/RollbarThread.m

```diff
@@ -43,7 +45,9 @@ - (void)checkItems {
     }
     
     @autoreleasepool {
-        RollbarSdkLog(@"Checking items...");
+        if ((nil != _logger) && (NO == _logger.configuration.developerOptions.suppressSdkInfoLogging)) {
```

Comment:
```suggestion
        if (_logger && !_logger.configuration.developerOptions.suppressSdkInfoLogging) {
```
or to increase readability (IMHO it's not necessary here as it's readable anyway):

```suggestion
        if (_logger && (_logger.configuration.developerOptions.suppressSdkInfoLogging == NO)) {
```

Reply:
same as above

Reply:
It's not the same though. I can agree on the part discussed above but I would like to hear your justification for the `nil != _logger` expression before accepting it.

Here is my final suggestion:

```suggestion
        if (_logger && (NO == _logger.configuration.developerOptions.suppressSdkInfoLogging)) {
```

Reply:
@bxsx , let's just start from [Apple's Objective-C documentation](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithObjects/WorkingwithObjects.html#//apple_ref/doc/uid/TP40011210-CH4-SW22) and search the page for "Working with nil"....

Reply:
@akornich good point!
Could you point me to the `nil != object` example in the documentation? I'm only seeing the ones that I've already suggested ;)